am-lib/src/main/resources/db/migrationAks/V19__load_data.sql

```diff
@@ -0,0 +1,9 @@
+insert into services (service_name, service_description) values ('cmc-test', null)
+on conflict on constraint services_pkey do nothing;
```

Comment:
Weren't we trying not to put any values into the DB in the flyway scripts?

Reply:
Yes..Fly way is only applicable for Local and AKS. And will not run in AAT as aat used Jenkins pipe line where we ar only using default migration location.